package.json

```diff
@@ -27,31 +26,47 @@
   },
   "license": "MIT",
   "dependencies": {
```

Comment:
Would probably be a good idea to pin at least the dev dependencies, if not all deps.